demo/mkldnn_quant/quant_aware/PaddleCV_mkldnn_quantaware_tutorial.md

```diff
@@ -0,0 +1,220 @@
+
+```
+config = {
+         'weight_quantize_type': 'channel_wise_abs_max',
+         'activation_quantize_type': 'moving_average_abs_max',
+         'quantize_op_types': ['depthwise_conv2d', 'mul', 'conv2d', 'elmentwise_add', 'pool2d']
+     }
+```
+If you want to understand the meaning of each parameter, please refer to [PaddleSlim quant_aware API](https://paddlepaddle.github.io/PaddleSlim/api/quantization_api/#quant_aware)
+
+**Note:**
+- Because the quantification needs to modify the Program, some training strategies that will modify the Program need to be closed. `` sync_batch_norm '' and quantized Doka training will be wrong when used at the same time, the reason is unknown, please make sure to close it in the code. (Closed in the sample code)
+```
+build_strategy.fuse_all_reduce_ops = False
+build_strategy.sync_batch_norm = False
+```
+- In `train_image_classification.py`, `paddleslim.quant.convert` is mainly used to change the order of quantization op and inverse quantization op in Program. In addition, `paddleslim.quant.convert ` will also change the operator parameters such as `conv2d`,` depthwise_conv2d`, and `mul` to the values within the range of quantized `int8_t`, but the data type is still `float32`. This is the qat float32 model we need, the default position is `./quantization_models/act_*/float `.
+
+## 3. Convert fp32 qat model to MKL-DNN INT8 model
+The model saved after training in the previous step is the float32 qat model. We also need to remove the quantization, inverse quantization op, fuse some ops, and fully converted into INT8 model. Run the following script
+
+```
+python ./save_qat_model.py --qat_model_path=$PATH_TO_float32_qat_dir --int8_model_save_path=$PATH_TO_SAVE_INT8_MODEL --quantized_ops="conv2d,pool2d"
+```
+
+## 4. Inference test
+
+### 4.1 Data preprocessing
+To run the inference test, the data needs to be converted to binary first. Run the script without any pramaters to transform the complete ILSVRC2012 val data set to bianary file. Use `local` parameter to transform your own data.
+```
+python ../tools/full_ILSVRC2012_val_preprocess.py --local --data_dir=$USER_DATASET_PATH --output_file=data.bin  
+```
+
+Optional parameters:
+- No parameters set. The script will download the ILSVRC2012_img_val data set and convert it into a binary file.
+- local, set to true, indicating that users will provide their own data
+- data_dir, set the user's own data directory
+- label_list, set the image path-image category list, similar to `val_list.txt`
+- output_file, the name of the generated bin file
+- data_dim, the length and width of the preprocessed image, the default is 224. Not recommended to change
+
+The user's own data set directory structure should be as follows
+```
+imagenet_user
+├── val
+│ ├── ILSVRC2012_val_00000001.jpg
+│ ├── ILSVRC2012_val_00000002.jpg
+| | ── ...
+└── val_list.txt
+```
+Among them, the content of val_list.txt should be as follows:
+```
+val / ILSVRC2012_val_00000001.jpg 0
```

Comment:
Are whitespaces around the `/` character required?

Reply:
Done